docs/ops/arithmetic/Erf_1.md

```diff
@@ -13,17 +13,15 @@ For each element from the input tensor calculates corresponding element in the o
 erf(x) = \pi^{-1} \int_{-x}^{x} e^{-t^2} dt
 \f]
 
-**Attributes**:
-
-    No attributes available.
+**Attributes**: *Erf* operation has no attributes.
 
 **Inputs**
 
-* **1**: A tensor of type T. **Required.**
+* **1**: A tensor `x` of type `T` and arbitrary shape. **Required**.
```

Comment:
As I see we have a different agreement for highlighting  `T`.
You can find it here: https://github.com/openvinotoolkit/openvino/pull/5618/

@lazarevevgeny Please have a look.

Reply:
Who we should mark `required` (where put the dot):
1) `**Required.**`
2) `**Required**.`
in docs/ops/ more often we use 1st approach 180/38

Reply:
We should wrap types with `*` characters. 

Regarding the "Required". I personally like the `**Required**.` approach, but we can ask @openvinotoolkit/openvino-docs-maintainers  on what is the better option.

Reply:
Both option 1, 2 fromat exist in current doc code,  I will choose the majority.